product/roadmap.md

```
@@ -0,0 +1,106 @@
+| F27 | Coach-side share-link experience polish             | 1d       | J7                   | Optimized read-only view for coach                       |
+| F28 | Garmin calendar grid view                           | 2d       | J2                   | Beyond the simple "tomorrow" card                        |
+| F29 | Recovery-adjusted glycemic-variability stats        | 1d       | J5                   | Cross-source insight #5                                  |
+| F30 | Light-theme parity polish                           | 2d       | UX                   | v0.1 ships dark-first                                    |
+
+---
+
+## Later (v0.3+ / candidates)
+
+| #                                                                   | Initiative                                  | Why later                    | Revisit when |
+| ------------------------------------------------------------------- | ------------------------------------------- | ---------------------------- | ------------ |
+| Coach multi-athlete dashboard                                       | Tertiary persona; v0.1 share-link is enough | First real coach asks        |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)